containers/cpp-mariadb/test-project/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+cd $(dirname "$0")
+
+source test-utils.sh vscode
+
+# Run common tests
+checkCommon
+
+# Run definition specific tests
+checkExtension "ms-vscode.cpptools"
+checkOSPackages "command-line-tools" build-essential cmake cppcheck valgrind clang lldb llvm gdb
+check "g++"  g++ -g main.cpp -o main.out -lmariadbcpp
+rm main.out
```

Comment:
Looks like `main.out` is not executed after build. This way db connectivity would be checked.
```sh
vscode ➜ /workspace/test-project $ ./main.out 
Hello, Remote World!
DB Connecting
DB Executing
Cluster has the following user created databases
mariadb
DB Success
```